modules/post-processing/yolo-object-detection/deployspec.yaml

```diff
@@ -0,0 +1,34 @@
+deploy:
+  phases:
+    install:
+      commands:
+      - npm install -g aws-cdk@2.20.0
+      - pip install -r requirements.txt
+    build:
+      commands:
+      - cdk deploy --require-approval never --progress events --app "python app.py" --outputs-file ./cdk-exports.json
+      # Here we export some env vars
+      - export ADDF_MODULE_METADATA=$(python -c "import json; file=open('cdk-exports.json'); print(json.load(file)['addf-${ADDF_DEPLOYMENT_NAME}-${ADDF_MODULE_NAME}']['metadata'])")
+      - export ECR_REPO_NAME=$(echo ${ADDF_MODULE_METADATA} | jq -r ".EcrRepoName")
+      - export IMAGE_URI=$(echo ${ADDF_MODULE_METADATA} | jq -r ".ImageUri")
+      - export REPOSITORY_URI=$AWS_ACCOUNT_ID.dkr.ecr.$AWS_DEFAULT_REGION.amazonaws.com/$ECR_REPO_NAME
+      - export COMMIT_HASH=$(echo $CODEBUILD_RESOLVED_SOURCE_VERSION | cut -c 1-7)
+      - export IMAGE_TAG=${COMMIT_HASH:=latest}
+      - export BASE_IMAGE=$(echo ${ADDF_MODULE_METADATA} | jq -r ".BaseImage")
+      # - Building the Docker image(s)...
+      # - Logging into AWS Deep Learning ECR Repository to pull Pytorch Base Image
+      - aws ecr get-login-password --region $AWS_DEFAULT_REGION | docker login --username AWS --password-stdin 763104351884.dkr.ecr.$AWS_DEFAULT_REGION.amazonaws.com
```

Comment:
what account id is this?

Reply:
See above - https://github.com/aws/deep-learning-containers/blob/master/available_images.md